### ISSUE TITLE
Comm target_module

### ISSUE THREAD

**Issue by SylvainCorlay** [ipython/ipykernel#252]
The comms have a `target_module` trait that is not documented in the spec and not used by ipywidgets.

**Comment by takluyver** [comment]
This identifies a requirejs module to load the target from. The idea was that either side could specify a fully qualified target name for the function to run on the other side.

I don't know of any code using `target_module`, but mobilechelonian uses `view_module` at the widgets level, which works in a similar way.

**Comment by SylvainCorlay** [comment]
yes, `view_module` and `model_module` are ipywidgets things, but `target_module` seems to be some left-over from older code, of the pre-big-split (tm) era.

**Comment by SylvainCorlay** [comment]
cc @jasongrout

**Comment by takluyver** [comment]
I'd assume it was added at the same time as `view_module` and `model_module`, for symmetry.
